resources-build-logic/src/main/kotlin/apple-main-convention.gradle.kts

```diff
@@ -16,6 +16,15 @@ kotlin {
         val iosMain by getting {
             dependsOn(appleMain)
         }
+        val watchosMain by getting {
+            dependsOn(appleMain)
+        }
+        val watchos64Main by getting {
+            dependsOn(watchosMain)
+        }
+        val watchos32Main by getting {
+            dependsOn(watchosMain)
+        }
```

Comment:
this part will be configured in resources-build-logic/src/main/kotlin/multiplatform-library-convention.gradle.kts

Reply:
convention plugins was reworked